### ISSUE TITLE
URL variables

### ISSUE THREAD

**Issue by cerullo** [DNNCommunity/DNN.FormAndList#67]
How do you access a URL variable so it can be used in the paging navigation? I have it implemented on a job search site. It worked well until the last upgrade of the module and it broke. My issue is paging. I need to pull URL variables and put them in the paging links. (1,2,3, at bottom for page navigation) I can not figure out how to call the URL variables for these paging links??? EG - URL - http://www.intlmedicalplacement.com/Job-Search-NEW?st=AR&sp=Critical%20Care%20Medicine  and this is what the paging navigation link is which is missing the ST and SP variables - http://www.intlmedicalplacement.com/Job-Search-NEW?udt_543_param_page=3&st=&sp=

**Comment by valadas** [comment]
I am not super familiar with advanced usages like this unfortunately. Maybe @SCullman can chime in on this ?

**Comment by SCullman** [comment]
Please check here:
https://github.com/DNNCommunity/DNN.FormAndList/wiki/Token-Based-XSLT--Generator-and-Editor

TLDR - Strongly biased

- FnL **ignores** any query string params which don't follow the `"udt_" + moduleId+ param name` pattern
- It takes time to set it up correctly, and It takes a lot of time to debug third party scripts. 

So read the instructions carefully, setup a new module to learn how it works step by step or hire some one (Don't ask me). Or consider to use a better module like [2sxc](https://github.com/2sic/2sxc/) for this task.
